django/db/models/expressions.py

```diff
@@ -1486,16 +1521,10 @@ class Exists(Subquery):
     template = "EXISTS(%(subquery)s)"
     output_field = fields.BooleanField()
 
-    def __init__(self, queryset, negated=False, **kwargs):
```

Comment:
The `negated` kwarg is not documented, but I'd add a small note in the backward incompatibility section that it's removed from `When()` and `Exists()`.